pillar/fluentd/fluentd_log.sls

```diff
@@ -0,0 +1,15 @@
+fluentd:
+  configs:
+    - name: fluentd_log
```

Comment:
If we're applying these pillar files to the same host then the list items will override each other. To be able to split them out like this we'll need to change the formula to take pillar data in the format:

```
fluentd:
  configs:
    fluentd_log:
      settings:
         foo: bar
     math_all_tls:
       settings:
         baz: foo
```